src/main/java/org/gitlab4j/api/ServicesApi.java

```diff
@@ -493,4 +494,58 @@ public void deleteCustomIssueTrackerService(Object projectIdOrPath) throws GitLa
         delete(Response.Status.OK, null, "projects", getProjectIdOrPath(projectIdOrPath), "services", "custom-issue-tracker");
 
     }
+
+    /**
+     * Get the Custom Issue Tracker service settings for a project.
```

Comment:
Can you update javadoc accordingly to gitlab documentation? https://docs.gitlab.com/ee/api/services.html#emails-on-push